yt/testing.py

```diff
@@ -1190,7 +1189,7 @@ def skip(*args, **kwargs):
             msg = "`{}` backend not found, skipping: `{}`".format(backend, func.__name__)
             print(msg)
             pytest.skip(msg)
-        if yt._called_from_pytest:
+        if ytcfg.get("yt", "__withintesting"):
```

Comment:
I think the error comes from this, which will return `True` for *both* nose (which doesn't gloss over `pytest.skip()` calls) and pytest (which will *not* fail with `pytest.skip()` calls).

Reply:
In the triage we thought one option might be to add another keyword to the `ytcfg` for pytest-status.

Reply:
Changed to `__withinpytest`